pom.xml

```diff
@@ -37,7 +37,8 @@
     </modules>
 
     <properties>
-        <gaffer.version>1.16.0</gaffer.version>
+        <revision>1.17.1</revision>
+        <gaffer.version>1.17.1</gaffer.version>
```

Comment:
Why is there a `revision` and a `gaffer.version` variable? Are these not the same?

Reply:
`revision` is a maven keyword which allows us to use a variable to define the version of POM modules. Currently each module version matches the gaffer version however this has to be done manually. `gaffer.version` cannot be used for module versioning as it is a normal variable. Revision simplifies the process so it only needs to be changed in one place.